### PR TITLE
add basic i18n support

### PR DESCRIPTION
**Pull request by ofa-**
provided languages:
- en
- fr

default = en

### CODE REVIEW

**Comment by ofa-** [comment]
- can add GUI to select desired language in preferences
- can add automatic default lang selection based on Thunderbird's locale

**Comment by ofa-** [comment]
Ok.  I can extend this PR and add an extra drop-down in the prefs gui to select the desired language.  I am not there yet but seems quite easy.  For the automatic selection based on TB's locale, I currently have no clue.

Note: there is potentially more to come, e.g. optional mime-date (sender's date) support, extra 'Lookout' header, quote-char removal fix (I think).  See the commits on branch ofa-next on my repo.

**Comment by jeevatkm** [comment]
@ofa- I will have a look on your commits. Thanks.

File: content/rwh-prefs.js [hunk 1]
```diff
@@ -85,6 +85,10 @@ ReplyWithHeader.Prefs = {
     return this.getIntPref('header.lblseq.style');
   },
 
+  get headerLang() {
+    return this.getStringPref('header.lang');
```
Comment (jeevatkm) [inline content/rwh-prefs.js]: Please change from `header.lang` to `header.locale`

File: content/rwh.js [hunk 2]
```diff
@@ -917,6 +920,42 @@ var ReplyWithHeader = {
       this.Log.errorWithException('Unable to open RWH URL.', ex);
     }
   },
+
+};
+
+var i18n = {
```
Comment (jeevatkm) [inline content/rwh.js]: You chosen appropriate naming then before. Thanks.

File: defaults/preferences/rwh-defaults.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ pref('extensions.replywithheader.enable', true);
 pref('extensions.replywithheader.header.from.style', 1);
 pref('extensions.replywithheader.header.tocc.style', 1);
 pref('extensions.replywithheader.header.lblseq.style', 1);
+pref('extensions.replywithheader.header.lang', 'en');
```
Comment (jeevatkm) [inline defaults/preferences/rwh-defaults.js]: Please change from `extensions.replywithheader.header.lang` to `extensions.replywithheader.header.locale`

**Comment by ofa-** [comment]
Thanks for the merge.  As for updating the other "lang" stuff, I originally did a blunt `sed -i s/lang/locale/g` but re-read your comment and went on replacing only the refs you mentioned; sorry about the miss-understanding.

**Comment by jeevatkm** [comment]
Not to worry @ofa- . I'm glad to work along with your work. I have done those changes it available on master branch.

**Comment by ofa-** [comment]
part of #12

**Pull request merged**